mitodl/rapid-response-xblock
# Store responses to problems

#### What are the relevant tickets?
Fixes #3 

#### What's this PR do?
Adds a model to store problem submission events and changes the logger to use it

#### How should this be manually tested?
Make sure the app is installed by running `python setup.py install`. Run migrations. Then create a multiple choice problem and in LMS submit an answer for one of the choices. Look at `ProblemCheckRapidResponse` and verify that the values all look correct.
